prusa/link/web/wifi-wizard/src/Fetcher.svelte

```diff
@@ -0,0 +1,136 @@
+<script>
+    import { onMount } from "svelte"
+    import { requestUrl, probe, fetcher } from './utils.js'
+
+    const AP_FETCH_INTERVAL = 6000;
+    const INFO_FETCH_INTERVAL = 2000;
+    const PROBE_INTERVAL = 2500;
+
+    export let aps = [];
+    export let connectionDetails = [];
+    export let rawConnectionInfo = {};
+    export let probeDetails = [];
+    export let probeResults = [];
+
+    let isApSelected = false;
+    let selectedAp = {};
+
+    let probeTimeout = setTimeout(probeAll, PROBE_INTERVAL)
```

Comment:
timeout setup should be called in onMount

you can use onMount function more times in same component, to separate logic